Sources/ComposableArchitecture/Debugging/ReducerInstrumentation.swift

```diff
@@ -62,16 +62,15 @@ extension Effect where Error == Never {
     let sid = OSSignpostID(log: log)
 
     return self.on(
-      started: {
+      starting: {
         os_signpost(
-          .begin, log: log, name: "Effect", signpostID: sid, "%sStarted from %s", prefix,
-          actionOutput
+          .begin, log: log, name: "Effect", signpostID: sid, "%sStarted from %s", prefix, actionOutput
         )
       },
       completed: {
         os_signpost(.end, log: log, name: "Effect", signpostID: sid, "%sFinished", prefix)
       },
-      disposed: {
+      interrupted: {
```

Comment:
Cancelling in Combine is equivalent to disposing in ReactiveSwift. Also, even an interrupted Signal will eventually be disposed, but not every disposed Signal will be interrupted, so I think tracking `disposed` makes more sense.

Reply:
Thanks for clarification, I'll remove this change 👍